src/vcpkg/commands.add.cpp

```diff
@@ -45,22 +47,29 @@ namespace
                                  (msg::command_line),
                                  "",
                                  "'{command_line}' can only add one artifact at a time.");
-
 }
 
 namespace vcpkg::Commands
 {
     void AddCommand::perform_and_exit(const VcpkgCmdArguments& args, const VcpkgPaths& paths) const
     {
-        args.parse_arguments(AddCommandStructure);
+        (void)args.parse_arguments(AddCommandStructure);
         auto&& selector = args.command_arguments[0];
+
         if (selector == "artifact")
         {
             Checks::msg_check_exit(VCPKG_LINE_INFO,
                                    args.command_arguments.size() <= 2,
                                    msgAddArtifactOnlyOne,
                                    msg::command_line = "vcpkg add artifact");
```

Comment:
Daniel pointed out that the ce's `add` does accept multiple arguments.
We should fix this behavior in a different PR.